templates/default/ckan_properties.ini.erb

```diff
@@ -69,7 +69,7 @@ sqlalchemy.url = postgresql://<%= node['datashades']['ckan_web']['dbuser'] %>:<%
 
 ckan.datastore.write_url = postgresql://<%= node['datashades']['ckan_web']['dbuser'] %>:<%= node['datashades']['postgres']['password'] %>@<%= node['datashades']['version'] %>pg.<%= node['datashades']['tld'] %>/<%= node['datashades']['ckan_web']['dsname'] %>
 ckan.datastore.read_url = postgresql://<%= node['datashades']['ckan_web']['dsuser'] %>:<%= node['datashades']['postgres']['password'] %>@<%= node['datashades']['version'] %>pg.<%= node['datashades']['tld'] %>/<%= node['datashades']['ckan_web']['dsname'] %>
-
+ckanext.xloader.jobs_db.uri = postgresql://<%= node['datashades']['ckan_web']['dbuser'] %>:<%= node['datashades']['postgres']['password'] %>@<%= node['datashades']['version'] %>pg.<%= node['datashades']['tld'] %>/<%= node['datashades']['ckan_web']['dsname'] %>
```

Comment:
You may need to create another table space for xloader to look after
step 6 https://github.com/ckan/ckanext-xloader

If it is a production server, you'll want to store jobs info in a more robust database than the default sqlite file:
sudo -u postgres createdb -O ckan_default xloader_jobs -E utf-8
And add this list to the config:

ckanext.xloader.jobs_db.uri = postgresql://ckan_default:pass@localhost/**xloader_jobs**

Reply:
I was assuming it could reuse the datastore table space. Any particular reason why that wouldn't work?

Creating a third database is certainly *possible* but what is the benefit?

Reply:
https://github.com/ckan/ckanext-xloader/issues/17 suggests that jobs could actually be stored in the *main* database. Looks like it just creates a single table, "jobs", with one row per XLoader job. I don't think there's a significant danger of this clashing with another extension that wants to create a 'jobs' table.

Reply:
Since it creates only one table, it should be find unless the same table name is used by the other schema.

Reply:
thanks for doing the research